azure-datalake/analytics/pom.xml

```diff
@@ -0,0 +1,144 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.microsoft.azure</groupId>
+        <artifactId>azure-arm-parent</artifactId>
+        <version>0.0.2-beta</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+    <artifactId>azure-datalake-analytics</artifactId>

```

Comment:
>azure-datalake-analytics [](start = 16, length = 24)

In azure-libraries-for-java repo, the artifact id is `azure-mgmt-datalake-analytics`. I am not sure whether I should keep `azure-mgmt-datalake-analytics` or change it to `azure-datalake-analytics`